lms/lmsweb/views.py

```diff
@@ -1,34 +1,29 @@
-import os
 from functools import wraps
 from typing import Optional
 from urllib.parse import urljoin, urlparse
 
 import arrow  # type: ignore
-from flask import (
-    jsonify, make_response, render_template,
-    request, send_from_directory, url_for,
-)
+from flask import (jsonify, make_response, render_template, request,
```

Comment:
Please keep our sorting conventions in regards to `import`s.

It should be organized from a to z, and split into 3 blocks: standard modules,  3rd party modules and our modules.